README.md

```diff
@@ -5,7 +5,7 @@
 Project requirements can be found [here](https://github.com/microverseinc/curriculum-final-capstone/blob/main/projects/business_requirements.md)
 
 ## Kanban Board
-We are two members in team 
+We are two members in the team 
```

Comment:
- [ ] Some tests are not passing:

![image](https://user-images.githubusercontent.com/57421823/206867434-790df195-a6ad-45b6-b343-34496591f7d2.png)

Kindly check them to ensure your apps integrity. 👍